native/acter/src/api/room.rs

```diff
@@ -147,8 +148,21 @@ impl Room {
             bail!("Can't send typing notice to a room we are not in")
         };
 
+        let my_id = room
+            .client()
+            .user_id()
+            .context("User not found")?
+            .to_owned();
+
         RUNTIME
             .spawn(async move {
+                let member = room
+                    .get_member(&my_id)
```

Comment:
if we need to fetch `member` very often here, how about we just store it right in the `Room` object? It might have to be an `Option` as we could be facing looking at rooms, we aren't part of, but in general, that could save us quite a bit of noise and db-fetching in between...